LithoStrings/Classes/NSAttributesReturnFunctions.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  NSAttributesReturnFunctions.swift
+//  LithoStrings
+//
+//  Created by Remmington Damper on 1/26/22.
+//
+
+import Foundation
+import UIKit
+
+    public func foregroundColorReturner(color: UIColor) -> [NSAttributedString.Key: Any] {
```

Comment:
Once again, let's indent these back

Reply:
Let's also change the naming one more time. I know we can't just take off the Returner off of all of these, but having that be in the function names doesn't seem right. Let's call this one, for example, `foregroundColorAttr` so we make it clear that we're not just returning the foregroundColor, we're returning the attributes for a string with that foreground color